pyproject.toml

```diff
@@ -26,6 +26,28 @@ classifiers = [
 urls = { Homepage = "https://github.com/Marker-Inc-Korea/AutoRAG" }
 dynamic = ["version", "dependencies"]
 
+[tool.poetry]
+name = "AutoRAG"
+version = "v0.1.0"
+description = "Automatically Evaluate RAG pipelines with your own data. Find optimal structure for new RAG product."
+authors = ["Marker-Inc <vkehfdl1@gmail.com>"]
+readme = "README.md"
+license = "MIT"
```

Comment:
Our license is Apache 2.0 license

Reply:
I removed all attributes except the minimum required for poetry shell to run, as other information is included in the project section.